scripts/src/javatests/org/oppia/android/scripts/maven/GenerateMavenDependenciesListTest.kt

```diff
@@ -357,6 +359,10 @@ class GenerateMavenDependenciesListTest {
     )
   }
 
+  private fun initiazeCommandExecutorWithLongProcessWaitTime(): CommandExecutorImpl {
+    return CommandExecutorImpl(processTimeout = 300, processTimeoutUnit = TimeUnit.SECONDS)
```

Comment:
Consider just setting it to 5 minutes instead--that's one of the benefits of being able to specify time unit.

Reply:
Done.

Reply:
Done.